pkgs/development/libraries/libsystemtap/default.nix

```diff
@@ -0,0 +1,27 @@
+{stdenv, fetchgit, gettext, python, elfutils}:
```

Comment:
can we use python3?

Reply:
Unfortunately, doesn't look like it.

`configure: error: python version 2 is required`